src/main/java/com/ethanyidong/bunny/Bunny.java

```diff
@@ -0,0 +1,82 @@
+package com.ethanyidong.bunny;
+
+import com.ethanyidong.bunny.arg.InvalidCommandException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Bunny {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        BunnySession bunny = new BunnySession();
+
+        bunny.printMessage("Hello! I'm Bunny.\nWhat can I do for you?");
+
+        replloop:
+        while (true) {
+            String input = in.nextLine();
+            ParsedCommand inputCommand = new ParsedCommand(input);
+
+            try {
+                inputCommand.validateCommand(bunny);
+            } catch (InvalidCommandException ice) {
+                bunny.printMessage(ice.toString());
+                continue;
+            }
+
+            switch (inputCommand.getCommand()) {
```

Comment:
Really nice :) I love that you are using `switch`/`case` statements here, as it helps a lot in readability compared to `if`/`else` statements.